docs/nasa_exoplanet_archive/nasa_exoplanet_archive.rst

```diff
@@ -6,56 +6,184 @@
+    >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
+    >>> NasaExoplanetArchive.query_region(
+    ...     "exoplanets",
+    ...     SkyCoord(ra=172.56 * u.deg, dec=7.59 * u.deg),
+    ...     1.0 * u.deg
+    ... )
+    <QTable masked=True length=2>
+    pl_hostname pl_letter pl_name ...   dist     angle         sky_coord
+                                  ...                           deg,deg
+        str5       str1     str7  ... float64   float64          object
+    ----------- --------- ------- ... -------- ---------- -------------------
+          K2-18         b K2-18 b ... 6.016586 164.332463 172.560455,7.588391
+          K2-18         c K2-18 c ... 6.016586 164.332463 172.560455,7.588391
+
+
+The most general queries can be performed using the `~astroquery.nasa_exoplanet_archive.NasaExoplanetArchiveClass.query_criteria` method.
+For example, a full table can be queried as follows:
 
 .. code-block:: python
 
-        >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
-        >>> hatp11b = NasaExoplanetArchive.query_planet('HAT-P-11 b')
+    >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
+    >>> NasaExoplanetArchive.query_criteria("cumulative", select="*")
+    <QTable masked=True length=9564>
+     kepid   kepoi_name kepler_name  ... koi_fittype koi_score      sky_coord
+                                     ...                             deg,deg
+     int64      str9       str15     ...     str7     float64         object
+    -------- ---------- ------------ ... ----------- --------- -------------------
+    10797460  K00752.01 Kepler-227 b ...     LS+MCMC       1.0 291.93423,48.141651
+    10797460  K00752.02 Kepler-227 c ...     LS+MCMC     0.969 291.93423,48.141651
+         ...        ...          ... ...         ...       ...                 ...
+    10155286  K07988.01           -- ...     LS+MCMC     0.092 296.76288,47.145142
+    10156110  K07989.01           -- ...     LS+MCMC       0.0 297.00977,47.121021
 
 
-Properties of a particular planet
-=================================
+Example queries
+===============
 
-The properties of each planet are stored in a table, with `columns defined by
-the NASA Explanet Archive <https://exoplanetarchive.ipac.caltech.edu/docs/API_exoplanet_columns.html>`_.
-There is also a special column of sky coordinates for each target, named
-``sky_coord``.
+Specific searches can be executed using the ``where``, ``select``, ``order``, and other parameters as described in the documentation [1]_.
+
+In this section, we demonstrate
+
+1. The number of confirmed planets discovered by TESS:
 
 .. code-block:: python
 
-        >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
-        >>> hatp11b = NasaExoplanetArchive.query_planet('HAT-P-11 b')
+    >>> from astroquery.nasa_exoplanet_archive import NasaExoplanetArchive
+    >>> NasaExoplanetArchive.query_criteria(
+    ...     "exoplanets",
```

Comment:
it might be nice to use the explicit keyword argument here (i.e., `table="exoplanets"`)

Reply:
Do you mean just in the example or explicitly turn this into a required kwarg in the implementation?

Reply:
Just in the example.  At a glance, I wasn't sure what "exoplanets" meant in context.

Reply:
👍

Reply:
also done!